app/components/store-concept-scheme-selector.js

```diff
@@ -0,0 +1,50 @@
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+import { action } from '@ember/object';
+import { queryDB } from '../utils/query-sparql-query';
+
+export default class StoreConceptSchemeSelectorComponent extends Component {
```

Comment:
WE should be creating the custom components for the radio & checkbox selectors to if we implement the overwrite method for the preview.